src/main/java/com/smartbear/readyapi/client/execution/CodegenBasedTestServerApi.java

```diff
@@ -328,6 +331,9 @@ public ProjectResultReport postSwagger(URL swaggerApiURL, String endpoint, boole
         List<Pair> queryParams = new ArrayList<>();
         queryParams.add(new Pair("async", String.valueOf(async)));
         queryParams.add(new Pair("endpoint", endpoint));
```

Comment:
True, @veinSmart can you please add the null check for endpoint also while you are at it ?